pkg/logging/logging.go

```diff
@@ -50,3 +50,12 @@ func New(w io.Writer, level logrus.Level) Logger {
 func (l *logger) NewEntry() *logrus.Entry {
 	return logrus.NewEntry(l.Logger)
 }
+
+// ToLogLevel bounds a numeric loglevel value to the amount of loglevels
+// provided by the underlying logging package
+func ToLogLevel(level int) logrus.Level {
```

Comment:
This function is aways called before `logging.New`. It can be implicit in `logging.New` to protect the log level without the need to be called explicitly and also should be unexported.